src/domotica/domotica_fastclock.c

```diff
@@ -83,25 +85,26 @@ void fast_clock_handle_update(FAST_CLOCK_TIME_Type time){
 
   for(uint8_t index = 0 ; index < DOMOTICA_FASTCLOCK_SIZE ; index++)
   {
-    // If last_timestamp is smaller than the current_time, we just check
-    // whether the timestamp is in interval (last_timestamp, current_timestamp]
-    // Otherwise, i.e., last_timestamp is bigger than current_time, then we
-    // need to check whether the timestamp is in (last_timestamp, 2400) or
-    // in [0, current_timestamp]. Hence the huge if condition.
-    if (
-          (   last_timestamp < current_time
-           && timestamps[index].timestamp > last_timestamp
-           && timestamps[index].timestamp <= current_time
-          )
-          ||
-          (   last_timestamp > current_time
-           && (   (   timestamps[index].timestamp > last_timestamp
-                   && timestamps[index].timestamp < 2400
-                  )
-               || timestamps[index].timestamp <= current_time
-              )
-          )
-       )
+    if (timestamps[index].timestamp >= 2400)
```

Comment:
Op dit moment hebben we geen enabled/disabled LNCV per fast_clock entry. De enige manier om hem dus "snel" uit te zetten, is door de timestamp ongeldig te maken. Vandaar dat ik de controle in iedere tick doe...